code/modules/projectiles/projectile/bullets/sniper.dm

```diff
@@ -1,56 +1,97 @@
-// .50 (Sniper)
+// .50 BMG (Sniper)
 
 /obj/projectile/bullet/p50
-	name =".50 bullet"
+	name =".50 BMG bullet"
 	speed = 0.4
 	range = 400 // Enough to travel from one corner of the Z to the opposite corner and then some.
 	damage = 70
 	paralyze = 100
 	dismemberment = 50
+	catastropic_dismemberment = TRUE
 	armour_penetration = 50
+	///If TRUE, deal additional damage to objects.
 	var/breakthings = TRUE
+	///Determines how much additional damage the round does to mechs. For most rounds, needs breakthings to be TRUE.
+	var/mecha_damage = 30
 
 /obj/projectile/bullet/p50/on_hit(atom/target, blocked = 0)
 	if(isobj(target) && (blocked != 100) && breakthings)
-		var/obj/O = target
-		O.take_damage(80, BRUTE, BULLET, FALSE)
+		var/obj/thing_to_break = target
+		var/damage_to_deal = damage * 2
+		if(ismecha(thing_to_break))
+			damage_to_deal += mecha_damage
```

Comment:
with the extra damage this actually does 140+30=170 damage, which is more than the disruptor (without EMP effects), so that might be a little bonkers. Just take that into consideration.